packages/utils/src/lib/audio.ts

```diff
@@ -63,31 +74,29 @@ export const usePlayAudioUrl = () => {
       // if not currently selected song, stop playing
       if (audio?.playing()) {
         audio.stop();
+        dispatch(resetAudioState());
```

Comment:
I was having some issues with order of operations for the Howl `onstop` callback when switching between song previews, but will revisit one more time to see if I can refactor the state resets.